packages/tasit-identity-contract/src/GnosisSafe.test.js

```diff
@@ -119,94 +128,106 @@ describe("GnosisSafe", () => {
       );
       await execTxAction.waitForNonceToUpdate();
 
-      const balanceAfterWithdraw = await provider.getBalance(
-        GNOSIS_SAFE_ADDRESS
-      );
-      expect(`${balanceAfterWithdraw}`).to.equal(`${ZERO}`);
+      const balance = await provider.getBalance(GNOSIS_SAFE_ADDRESS);
+      expect(`${balance}`).to.equal(`${ZERO}`);
     });
+  });
 
-    it("wallet owner should withdraw some ethers", async () => {
+  describe("test cases that needs ERC20 deposit to the wallet", async () => {
```

Comment:
I suppose we might also have some test cases that need ERC20 deposit to a "user" who isn't the contract-based account/wallet.

Reply:
Agree